functions.c

```diff
@@ -640,7 +641,6 @@ const struct MenuFuncOp OpPager[] = { /* map: pager */
 #ifdef USE_NNTP
   { "reconstruct-thread",            OP_RECONSTRUCT_THREAD },
 #endif
-  { "redraw-screen",                 OP_REDRAW },
```

Comment:
I think we need this.
Because the Pager doesn't use the generic menu, it needs its own "redraw-screen".